app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'IpCamera';
+
+const p = Object.freeze({
+  // Public getter only properties.
+  manufacturer: Symbol('manufacturer'),
+  model: Symbol('model'),
+  name: Symbol('name'),
+
+  // Private getter only properties.
+  adapter: Symbol('adapter'),
+  getters: Symbol('getters'),
+  setters: Symbol('setters'),
+});
+
+export default class IpCameraService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+
+    // Public getter only properties.
+    this[p.manufacturer] = props.properties.manufacturer;
+    this[p.model] = props.properties.model;
+    this[p.name] = props.properties.name;
+
+    // Private getter only properties.
+    this[p.adapter] = props.adapter;
```

Comment:
I thought it's easier to read and we only keep the properties we need (better code hygiene) but I don't have strong opinion.
If you strongly disagree, I'll adopt the second approach like `this[p.service].getters`.